### ISSUE TITLE
Translation and remove AM/PM

### ISSUE THREAD

**Issue by FierceNorse** [avirakesh/bedside-pi#1]
Hi,

I love this project, and have it running on my RPI3B with the 7" screen connected on my nightstand.

I live in Norway, and we always use the 24H clock. Is it possible to remove the AM/PM indicator to the right of the time, and always just display the 24H time? 
Also, is it possible to change the names for the forecast (e.g. changing "Partly cloudy" to "Delvis skyet"), and the names of days under the clock? 

If it's just changing some parameters in the config, please just point me in the right direction. 

Thank you for sharing your work 👍

**Comment by avirakesh** [comment]
Hi,

Glad you're enjoying the project. 
I have updated the the project to include multiple languages for weather, and 24 hr clock.

The README has been updated to reflect the same.

Language can be changed in **modules/prefs.js** and 24 hr clock can be toggled in **assets/js/script.js** 

Feel free to let me know if you run into any issues.

**UPDATE:**
There is now an array in assets/js/scripts.js that contains the name of days and months. Change them according to your needs
